### ISSUE TITLE
test_ident uses deprecated function log.flushErrors

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2767]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#2767|
|Type|defect|
|Created|2007-08-02 11:55:19Z|

Let's remove this.

<details><summary>Searchable metadata</summary>

```
trac-id__2767 2767
type__defect defect
reporter__therve therve
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1186055719000000 1186055719000000
changetime__1192949362000000 1192949362000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve commented|
|-|-|

(In [20919])  
 * Replace call to log.flushErrors
 * 80 cols, trailing whispaces
 * Unused import
 * Refactor one of the test: split it, and use assertFailure instead of relying
 on the synchoncity of the response.

Refs [#2767](https://github.com/twisted/twisted/issues/2767)

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve commented|
|-|-|

Branch is test-ident-cleanups-2767.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)|@therve removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @therve|
|-|-|

Flawless

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [21374]) Merge test-ident-cleanups-2767

Author: therve
Reviewer: exarkun
Fixes [#2767](https://github.com/twisted/twisted/issues/2767)

Replace a call of log.flushErrors by flushLoggedErrors in
twisted.test.test_ident, and make a bunch of cleanups in the process.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
